templates/templated_email/account/account_delete.email

```diff
@@ -7,7 +7,7 @@
 {% block plain %}
 {% include 'templated_email/shared/_header.email' %}
 {% blocktrans context "Account delete reset e-mail text" %}
-You're receiving this e-mail because you or someone else has requested a deletion of your user account at {{ domain }}.
+You're receiving this e-mail because you or someone else has requested a deletion of your user account at {{ site_mname }}.
```

Comment:
Typo in `site_mname`.

Reply:
Is this a typo only in the template or is it wrong in the context in the function that sends the email?

Reply:
typo in template, all of the emails get the context from a single function called `get_email_context` that provides `site_name` (resulting context is ocasionally extended later).